testgrid/config/config.yaml

```diff
@@ -1400,6 +1400,53 @@ dashboards:
   - name: gke-master
     test_group_name: ci-kubernetes-e2e-gke-latest-upgrade-master
 
+- name: sig-cli
+  dashboard_tab:
+  - name: gce
+    test_group_name: ci-kubernetes-e2e-gce
+    base_options: 'include-filter-by-regex=Kubectl%20client'
+    description: 'kubectl e2e tests for master branch'
+  - name: gce-1.6
+    test_group_name: ci-kubernetes-e2e-gce-release-1.6
+    base_options: 'include-filter-by-regex=Kubectl%20client'
+    description: 'kubectl e2e tests for 1.6 branch'
+  - name: gce-1.5
+    test_group_name: ci-kubernetes-e2e-gce-release-1.5
+    base_options: 'include-filter-by-regex=Kubectl%20client'
+    description: 'kubectl e2e tests for 1.5 branch'
+  - name: gce-1.4
+    test_group_name: ci-kubernetes-e2e-gce-release-1.4
+    base_options: 'include-filter-by-regex=Kubectl%20client'
+    description: 'kubectl e2e tests for 1.4 branch'
+  - name: gce-flaky
+    test_group_name: ci-kubernetes-e2e-gce-flaky
+    base_options: 'include-filter-by-regex=Kubectl%20client'
+    description: 'kubectl flaky e2e tests for master branch'
+  - name: gce-slow
+    test_group_name: ci-kubernetes-e2e-gce-slow
+    base_options: 'include-filter-by-regex=Kubectl%20client'
+    description: 'kubectl slow e2e tests for master branch'
+  - name: gce-serial
+    test_group_name: ci-kubernetes-e2e-gce-serial
+    base_options: 'include-filter-by-regex=Kubectl%20client'
+    description: 'kubectl serial e2e tests for master branch'
+  - name: unit
+    test_group_name: ci-kubernetes-test-go
+    base_options: 'include-filter-by-regex=pkg/kubectl'
+    description: 'unit and integration tests for master branch'
```

Comment:
This is not true. The test-go jobs run `make test`, `make test-cmd` and `make test-integration`. The verify jobs do not run `make test-cmd`

The test-go jobs eventually run `hack/jenkins/test-dockerized.sh` which in turn runs `make test-cmd` ([link](https://github.com/kubernetes/kubernetes/blob/master/hack/jenkins/test-dockerized.sh#L59))


test-go jobs https://github.com/kubernetes/test-infra/blob/master/jobs/config.json#L162
run the verify scenario without a `--script` flag, which defaults to `hack/jenkins/test-dockerized.sh` https://github.com/kubernetes/test-infra/blob/master/scenarios/kubernetes_verify.py

By contrast the verify jobs https://github.com/kubernetes/test-infra/blob/master/jobs/config.json#L199 
run the verify scenario with a `--script=hack/jenkins/verify-dockerized.sh`, which only runs `make verify` [link](https://github.com/kubernetes/kubernetes/blob/master/hack/jenkins/verify-dockerized.sh)